L1Trigger/L1THGCal/plugins/geometries/HGCalTriggerGeometryHexImp1.cc

```diff
@@ -54,10 +54,10 @@ void HGCalTriggerGeometryHexImp1::fillMaps(const es_info& esInfo)
 {
     //
     // read module mapping file
-    std::map<short, short> wafer_to_module_ee;
-    std::map<short, short> wafer_to_module_fh;
-    std::map<short, std::map<short,short>> module_to_wafers_ee;
-    std::map<short, std::map<short,short>> module_to_wafers_fh;
+    std::unordered_map<short, short> wafer_to_module_ee;
+    std::unordered_map<short, short> wafer_to_module_fh;
+    std::unordered_map<short, std::unordered_map<short,short>> module_to_wafers_ee;
+    std::unordered_map<short, std::unordered_map<short,short>> module_to_wafers_fh;
```

Comment:
From the code below it looks like you can just use std::unordered_multimap<short, short>, which would be more compact in memory and faster. Any reason why not?